scripts/themes.json

```diff
@@ -0,0 +1,1113 @@
+[
+    {
+        "name": "AlabasterDark",
+        "display_name": "Alabaster Dark",
+        "parent_theme": "BaseTheme",
+        "is_light_theme": false,
+        "attribs": {
+            "colorKeyboard": "#1f1f1f",
```

Comment:
Where do the themes come from ?

Reply:
I just asked ChatGPT to generate all them haha